spec/models/embedded_ansible_worker/runner_spec.rb

```diff
@@ -10,7 +10,9 @@
     let(:runner) {
       worker
       allow_any_instance_of(described_class).to receive(:worker_initialization)
-      described_class.new(:guid => worker_guid)
+      r = described_class.new(:guid => worker_guid)
+      allow(r).to receive(:worker).and_return(worker)
```

Comment:
😢   I have to finish fixing this.  So much code now depends on all of this junk running when you call `.new` though.